models/models.topics.js

```diff
@@ -3,7 +3,13 @@ const db = require("../db/connection.js");
 exports.selectTopics = () => {
   const SQL = `SELECT * FROM topics;`;
 
-  return db.query(SQL).then((data) => {
-    return data;
+  return db.query(SQL).then(({ rows, rowCount }) => {
+    if (rowCount === 0) {
+      return Promise.reject({
```

Comment:
in theory for this one, as it is only selecting `*` from topics, rowCount should never be 0